spec/views/two_factor_authentication/otp_expired/show.html.erb_spec.rb

```diff
@@ -0,0 +1,9 @@
+require 'rails_helper'
+
+describe 'two_factor_authentication/otp_expired/show.html.erb' do
```

Comment:
Should we flesh out this spec a bit more, e.g. asserting that it has a link to the try again?

Reply:
Yes, that is the main piece that I am working on testing